browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -1379,10 +1379,28 @@ Are you sure you want to do this?
         New private tab
       </message>
       <message name="IDS_BRAVE_NEW_TAB_PRIVATE_HEADER" desc="Header shown when a user opens a private tab explaining private mode">
-        This is a Private Tab
+        Private tab
       </message>
-      <message name="IDS_NEW_TAB_PRIVATE_DESC" desc="Message shown when a user opens a private tab explaining private mode">
-        Even though sites you visit in private tabs are not saved locally, they do not make you anonymous or invisible to your ISP, your employer, or to the sites you are visiting.
+      <message name="IDS_BRAVE_NEW_TAB_PRIVATE_NO_TRACE" desc="No trace title shown when a user opens a private tab explaining private mode">
+        Leave no trace on your device
+      </message>
+      <message name="IDS_BRAVE_NEW_TAB_PRIVATE_NO_TRACE_DESCRIPTION" desc="No trace description shown when a user opens a private tab explaining private mode">
+        Sites you visit in private windows won't appear in your history. Cookies, form data, and site data vanish once all private windows are closed.
+      </message>
+      <message name="IDS_BRAVE_NEW_TAB_PRIVATE_VISIBILITY" desc="Browsing visibility title shown when a user opens a private tab explaining private mode">
+        Browsing visibility
+      </message>
+      <message name="IDS_BRAVE_NEW_TAB_PRIVATE_VISIBILITY_DESCRIPTION" desc="Browsing visibility description shown when a user opens a private tab explaining private mode">
+        Note that -even in private windows- your browsing activity may still be visible to your ISP, network observers, or your employer (if you're using a work machine/network).
```

Comment:
Should we say "device" instead of "machine" since this is only applicable to Android?

Reply:
Yes, I'm changing it!